test/test_math_ops.cpp

```diff
@@ -814,6 +814,56 @@ TEST(MathOps, BatchMatMul4DAdjXY) {
   opexecuter.RunTest();
 }  // end of test op BatchMatMul
 
+// Test op: BatchMatMulV2
+// BatchMatMul2D
+// AdjX = false
+// AdjY = false
+TEST(MathOps, BatchMatMulV22D) {
+  Scope root = Scope::NewRootScope();
+
+  Tensor A(DT_FLOAT, TensorShape({2, 3}));
+  Tensor B(DT_FLOAT, TensorShape({3, 4}));
+
+  AssignInputValues(A, 2.0f);
+  AssignInputValues(B, 7.0f);
+
+  vector<int> static_input_indexes = {};
+  auto R = ops::BatchMatMulV2(root, A, B);
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
+
+  std::vector<Output> sess_run_fetchoutputs = {R};
+  OpExecuter opexecuter(root, "BatchMatMulV2", static_input_indexes,
+                        output_datatypes, sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+}  // end of test op BatchMatMulV2
+
+// Test op: BatchMatMulV2
+// BatchMatMulV23D
+// AdjX = false
+// AdjY = false
+TEST(MathOps, BatchMatMulV23D) {
+  Scope root = Scope::NewRootScope();
+
+  Tensor A(DT_FLOAT, TensorShape({10, 2, 3}));
+  Tensor B(DT_FLOAT, TensorShape({10, 3, 4}));
+
+  AssignInputValues(A, 2.0f);
+  AssignInputValues(B, 7.0f);
+
+  vector<int> static_input_indexes = {};
+  auto R = ops::BatchMatMulV2(root, A, B);
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
+
+  std::vector<Output> sess_run_fetchoutputs = {R};
+  OpExecuter opexecuter(root, "BatchMatMulV2", static_input_indexes,
+                        output_datatypes, sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+}  // end of test op BatchMatMulV2
+
```

Comment:
Can we add more tests for BatchMatMulV2 like Adjx, Adjy and 4D  like we have for BatchMatmul https://github.com/tensorflow/ngraph-bridge/blob/master/test/test_math_ops.cpp#L517

Reply:
Done ;)